pom.xml

```diff
@@ -14,254 +14,266 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <groupId>com.ibm.eventstreams.connect</groupId>
-  <artifactId>kafka-connect-mq-source</artifactId>
-  <packaging>jar</packaging>
-  <version>1.3.2</version>
-  <name>kafka-connect-mq-source</name>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.ibm.eventstreams.connect</groupId>
+    <artifactId>kafka-connect-mq-source</artifactId>
+    <packaging>jar</packaging>
+    <version>1.3.2</version>
+    <name>kafka-connect-mq-source</name>
     <organization>
-    <name>IBM Corporation</name>
+        <name>IBM Corporation</name>
+        <url>http://ibm.com</url>
+    </organization>
     <url>http://ibm.com</url>
-  </organization>
-  <url>http://ibm.com</url>
-  <description>
+    <description>
       A Kafka Connect connector for copying data from IBM MQ into Apache Kafka.
   </description>
-  <licenses>
-    <license>
-      <name>Apache License 2.0</name>
-      <url>http://www.apache.org/licenses/LICENSE-2.0.html</url>
-      <distribution>repo</distribution>
-    </license>
-  </licenses>
-  <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-  </properties>
-  <dependencies>
-    <dependency>
-      <groupId>org.apache.kafka</groupId>
-      <artifactId>connect-api</artifactId>
-      <version>2.6.0</version>
-      <scope>provided</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.kafka</groupId>
-      <artifactId>connect-json</artifactId>
-      <version>2.6.0</version>
-      <scope>provided</scope>
-    </dependency>
+    <licenses>
+        <license>
+            <name>Apache License 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.html</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>connect-api</artifactId>
+            <version>3.4.0</version>
```

Comment:
Would a jar built using a newer version of Kafka still work in a Connect runtime using an older version of Kafka? 

(Apologies - I should know the answer to this!)

If yes - great. 
If not - we should at least update the README to specify the minimum version of Connect required to use the connector if we make this change.

Reply:
Yes, it does work with older version of kafka connect runtime. I was able to test it with kafka-2.6.2 and higher also having mq 9 or higher. I have updated the readme with the same.

Reply:
@dalelane ⬆️